src/bc_protocol.rs

```diff
@@ -10,7 +12,7 @@ use std::time::Duration;
 
 use Md5Trunc::*;
 
-mod connection;
+pub mod connection;
```

Comment:
I changed this to pub so that I could access it from MediaData but thinking about it now I could: Move media_packet.rs into bc_protocl/ and then I don't need this.

Would you prefer that?

Reply:
Done